homeassistant/components/plum_lightpad/strings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Plum Lightpad",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Fill in your Plum Cloud login information",
```

Comment:
This is not a title. Just remove it and it will be automatically filled with the manifest title.
```suggestion
```